### PR TITLE
Dockerfile strukture update

### PR DESCRIPTION
**Pull request by petracihalova**
it is not needed to copy only pipfiles and then all folder content
before
```
COPY ./Pipfile ./Pipfile.lock /usr/src/app/
COPY . /usr/src/app/
```

now
```
COPY . /usr/src/app/
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,8 +3,11 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal
 WORKDIR /usr/src/app
 
 ENV FLASK_RUN_HOST 0.0.0.0
+
 ENV BACKENDS_CONFIG_MAP=/etc/turnpike/backends.yml
-COPY ./Pipfile ./Pipfile.lock /usr/src/app/
```
Comment (astrozzc) [inline Dockerfile:7]: I vaguely remember the reason we only copy pipfile to there is: only if pipfile is updated we need to run the following install. Other files updates won't start again the install block.

Comment (lpichler) [reply Dockerfile:7]: @astrozzc I am sorry I was faster, @petracihalova can you take a look ?

Comment (astrozzc) [reply Dockerfile:7]: no worries, it won't break anything :) @lpichler

Comment (petracihalova) [reply Dockerfile:7]: thank you @astrozzc, I will revert this

**Pull request merged**